site/docs/layouts/_default/single.html

```diff
@@ -8,7 +8,7 @@
           {{ .TableOfContents }}
         </aside>
         <main id="main" role="main" class="markdown">
-          {{ if .Title }}<h1>{{ .Title }}</h1>{{ end }}
+          {{ if .Title }}<h1 class="title">{{ .Title }}</h1>{{ end }}
```

Comment:
This should probably be duplicated in `list.html`

Reply:
Yep, please do.  All `_index.md` files are rendered with `list.html`.

I think we should probably just make `list.html` a symlink to `single.html`.

Reply:
Revised and symlinked now.